common/src/main/java/org/apache/sedona/common/raster/RasterOutputs.java

```diff
@@ -74,6 +77,25 @@ public static byte[] asGeoTiff(GridCoverage2D raster) {
         return asGeoTiff(raster, null, -1);
     }
 
+    /**
+    * Creates a GeoTiff file with the provided raster. Primarily used for testing.
+     * @param raster The raster to be stored as .tiff
+     * @param filePath The path where the .tiff should be stored.
+     *
+     * @return true if file is created, otherwise throws IOException
+     * @throws IOException
+     * */
+    public static boolean toGeoTiff(GridCoverage2D raster, String filePath) {
```

Comment:
This function should be called `writeToDiskFile`. It takes a byte[] array as input and write the file to disk. This is a generic function that works with the result of asGeoTiff, asArcGrid, and so on.